flask_pydantic/core.py

```diff
@@ -11,6 +11,7 @@
     JsonBodyParsingError,
     ManyModelValidationError,
 )
+from .exceptions import ValidationError as MyValidationError
```

Comment:
Can you please try to come up with a better naming than `MyValidationError`?

Reply:
I renamed it to FailedValidation on import instead of MyValidationError.